SPECS/kata-containers/kata-containers.spec

```diff
@@ -52,15 +47,6 @@ Source0:        https://github.com/%{name}/%{name}/archive/refs/tags/%{version}.
 Source1:        https://github.com/%{name}/%{name}/releases/download/%{version}/%{name}-%{version}-vendor.tar.gz
 Source2:        50-kata
```

Comment:
do we have some background on this file? It seems like we need this specifically for Mariner? It do not see this or a similar file for the -cc package? is it possible this can be included on the kata-containers GH side? Just like to be educated here.

Reply:
It's only referenced at line 168 for CRI-O support, so we probably can get rid of it at some point.